lib/vacuum/request.rb

```diff
@@ -21,7 +21,8 @@ class Request
       'IN' => 'webservices.amazon.in',
       'IT' => 'webservices.amazon.it',
       'JP' => 'webservices.amazon.co.jp',
-      'US' => 'webservices.amazon.com'
+      'US' => 'webservices.amazon.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.